src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -43,6 +43,13 @@ public void OnSearchTextChanged(object sender, EventArgs e)
                 SearchTextChanged(this, e);
         }
 
+        public event EventHandler WorkspaceChanged;
```

Comment:
As described in the comment above, these new codes should be removed. Instead, introduce a new one called `RequestCloseSearchToolTip`, follow the example of the existing `RequestReturnFocusToSearch` method/event.